### ISSUE TITLE
Loss not reducing, high validation and test metric values

### ISSUE THREAD

**Issue by parth-shettiwar** [ULTR-Community/ULTRA#19]
I tried to run the code with DLA algorithm on Yahoo dataset. Following is the output attached. I am not sure of the following observation where I am getting almost constant training loss of about 4  (with each rank loss and exam loss as about 2), and high validation and testing metric values of more than 0.9. I did try to observe the parameter values of 2 models, which are actually updating. Also the loss is just fluctuating in range of 3.9 to 4.5 always. Is there something I should do with hyperparameters, have kept the default learning rate of 0.05 and selection_bias_cutoff = 10. This is with respect to the pytorch implementation of the code
<img width="388" alt="Screen Shot 2022-07-08 at 4 01 37 PM" src="https://user-images.githubusercontent.com/37413794/178080130-d336cbc4-a50f-4825-99a9-f2d856c2b029.png">
